contracts/libraries/SafeMath.sol

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: GPL-3.0
+
+pragma solidity >=0.6.0;
```

Comment:
Solidity 0.8 has built-in SafeMath, there is no need to include SafeMath library